package-parser/package_parser/processing/migration/_differ.py

```diff
@@ -0,0 +1,45 @@
+from abc import ABC, abstractmethod
+
+from package_parser.processing.api.model import Class, Function, Parameter, Result
+
+
+class AbstractDiffer(ABC):
+    @staticmethod
+    @abstractmethod
+    def diff_attributes(
```

Comment:
How about calling these methods `compare_xy` or `compute_xy_similarity`? Might be more accurate since we only get a float back instead of a list of changes.